requirements.txt

```diff
@@ -1,24 +1,29 @@
 alembic==1.8.1
 ascii_magic==2.3.0
--e git+https://github.com/abrichr/atomacos.git; sys_platform == 'darwin'
+atomacos @ git+https://github.com/abrichr/atomacos.git#egg=atomacos ; sys_platform == 'darwin'
 bokeh==2.4.3
 clipboard==0.0.4
 deepdiff[optimize]==6.3.0
 dictalchemy3==1.0.0
+drawsvg==2.1.1
 fire==0.4.0
+gpt4all
```

Comment:
Can you please add a version?

Reply:
In [7d563f9](https://github.com/MLDSAI/OpenAdapt/pull/198/commits/7d563f99148ff1ab65d6b6b58c78c386fb4fab12) I removed it as it should be in the GPT4All PR (and I will add the version there)